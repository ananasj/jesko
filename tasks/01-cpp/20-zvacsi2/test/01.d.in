18
3 29 30 5 46 4 14 64 98 17 11 49 64 78 65 54 97 73
7
