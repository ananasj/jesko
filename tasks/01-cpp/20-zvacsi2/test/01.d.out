10 36 37 12 53 11 21 71 105 24 18 56 71 85 72 61 104 80
